plugins/module_utils/dnac.py

```diff
@@ -1990,6 +1990,14 @@ def _exec(self, family, function, params=None, op_modifies=False, **kwargs):
                         self.logger.debug(bapi_error)
                         break
 
+        except exceptions.ApiError as e:
+            self.fail_json(
+                msg=(
+                    "An error occured when executing operation."
```

Comment:
Can we print family_name  & function_name?

Reply:
added.